string waypoint1
---
float64 distance
